bin/getInfoRippled.sh

```diff
@@ -0,0 +1,47 @@
+ #!/bin/bash
+
+export TMP_LOC="/tmp/ripple_info"
```

Comment:
Better, I think would be to not specify an absolute path: `TMP_LOC=$(mktemp -d --tmpdir ripple_info.XXXX)`. If the user somehow can't write to `/tmp`, they'll likely define `$TMPDIR`, which `mktemp` will use, and now the script doesn't need to worry about it. (No need for `sudo` either.)

In fact, the template argument is optional, so `mktemp -d --tempdir` will give you something anonymous like `/tmp/tmp.IAELgzoWOM`